xml_converter/generators/code_generator.py

```diff
@@ -387,33 +380,29 @@ def write_attribute (self, output_directory: str) -> None:
             if metadata[filepath]['type'] in ["MultiflagValue", "CompoundValue", "Enum"]:
```

Comment:
This should just be

```python
                if metadata[filepath]['type'] == "MultiflagValue":
                    ...
                elif metadata[filepath]['type'] == "CompoundValue":
                    ...
                elif metadata[filepath]['type'] == "Enum":
                    ...
                else:
                    continue
                ...
```
This way we dont need to define all the string values twice.

Reply:
Good point. I'll make that change. I didn't know about continue